lib/internal/util/inspect.js

```diff
@@ -1259,6 +1259,49 @@ function removeDuplicateErrorKeys(ctx, keys, err, stack) {
   }
 }
 
+function markNodeModules(ctx, line) {
+  let tempLine = '';
+  let nodeModule;
+  let pos = 0;
+  while (nodeModule = nodeModulesRegExp.exec(line)) {
+    // '/node_modules/'.length === 14
+    tempLine += line.slice(pos, nodeModule.index + 14);
+    tempLine += ctx.stylize(nodeModule[1], 'module');
+    pos = nodeModule.index + nodeModule[0].length;
+  }
+  if (pos !== 0) {
+    line = tempLine + line.slice(pos);
```

Comment:
```suggestion
    return tempLine + StringPrototypeSlice(line, pos);
```

Reply:
This still isn’t resolved.

Reply:
See https://github.com/nodejs/node/pull/41082#discussion_r762458168